tokio-trace/src/macros.rs

```diff
@@ -946,6 +976,43 @@ macro_rules! valueset {
     (@val $k:ident) => { None };
 }
 
+#[doc(hidden)]
+#[macro_export(local_inner_macros)]
+#[cfg(all(feature = "emit_log_always", not(feature = "emit_log_optional")))]
```

Comment:
It looks like if different dependencies enable these features together, this macro won't exist?

Reply:
whoops! good eye!